migrations/20200604111227_create_commit_result_table.ts

```diff
@@ -0,0 +1,17 @@
+import * as Knex from "knex";
+
+const tableName = "commit_result";
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.schema.createTable(tableName, (builder: Knex.TableBuilder) => {
+    builder.increments("id").primary();
+    builder.integer("commitId").unsigned().notNullable();
+    builder.string("rule").notNullable();
+    builder.text("message").notNullable();
```

Comment:
I see you are using both `string` and `text` in the migrations. We usually just use the TEXT datatype, so let's unify and use `text`.

Also, what's relationship between rule & message? It seems to me message can be derived from rule directly.

Reply:
`rule` is the identifier for the type of commit result. `message` is the more human-readable result text with more detailed information about the commit result. 


Ex: `rule`: "`header_max_len`" while `message`: "`Max length of commit headers is 50 characters. Your commit header has 70 characters`"

Reply:
As a rule of thumb, we should prepare for l10n of user-facing messages. Although I don't think we'll need it in this project, we can just use a jsonb for the message parameters instead. Something like `jsonb("data")`, so we can store like `data: {max_length: 50, actual_length: 70}`.

Reply:
yeah, this data might not even be used. The important thing is know what type of diagnosis it is and the score